packages/preview-middleware-client/types/sap.ui.comp.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'sap/ui/comp/smarttable' {
+    import SmartTable from 'sap/ui/comp/smarttable/SmartTable';
+    export interface SmartTableExtended extends SmartTable {
+        getVariantManagement: () => string;
+        getEntitySet: () => string;
```

Comment:
This method is already defined in `SmartTable`, no need to redefine it here.

Reply:
removed